pkgs/development/ocaml-modules/janestreet/buildOcamlJane.nix

```diff
@@ -24,6 +24,6 @@ buildOcaml (args // {
 
   buildPhase = "OCAML_TOPLEVEL_PATH=`ocamlfind query findlib`/.. make";
 
-  installPhase = "opaline -prefix $prefix -libdir $OCAMLFIND_DESTDIR ${name}.install";
+  installPhase = "opaline -prefix $prefix -libdir $OCAMLFIND_DESTDIR ${pname}.install";
 
```

Comment:
```suggestion
  installPhase = ''
    opaline -prefix $prefix -libdir $OCAMLFIND_DESTDIR ${pname}.install
  '';
```

Reply:
should i really remove the `$` before `prefix` and `OCAMLFIND_DESTDIR`?